src_python/yaafelib/_compat.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+#
+# Yaafe
+#
+# Copyright (c) 2009-2010 Institut Télécom - Télécom Paristech
+# Télécom ParisTech / dept. TSI
+#
+# Author : Benoit Mathieu
+#
+# This file is part of Yaafe.
+#
+# Yaafe is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Yaafe is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import, print_function
+
+import sys
+
+
+PY2 = sys.version_info[0] == 2
+PY3 = sys.version_info[0] == 3
```

Comment:
Only one is needed atm, and PY4 is not even being dreamed about yet.  If comparisons are always done using the same name, the runtime can optimise them.